.gitignore

```diff
@@ -2,4 +2,5 @@ node_modules
 package-lock.json
 build
 .env
+/frontend/packages
```

Comment:
This is no longer needed with the use of the volume mount

Reply:
removed